src/emails/templates/HeaderStyles.tsx

```diff
@@ -3,15 +3,28 @@
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
 // Universal email header styles
+export const MetaTags = () => {
+  return (
+    <mj-raw>
+      <meta name="color-scheme" content="light dark" />
+      <meta name="supported-color-schemes" content="light dark" />
+    </mj-raw>
+  );
+};
+
 export const HeaderStyles = () => {
   const hideBgImageOnDarkMode = `
+      :root {
+      color-scheme: light dark;
+      supported-color-schemes: light dark;
+      }
```

Comment:
Formatting: Indentation.

```suggestion
    :root {
      color-scheme: light dark;
      supported-color-schemes: light dark;
    }
```